code/__HELPERS/unsorted.dm

```diff
@@ -1416,3 +1416,24 @@ GLOBAL_PROTECT(valid_HTTPSGet)
 		mob_occupant = brain.brainmob
 
 	return mob_occupant
+
+/proc/msglog_admins(text)
+	message_admins(text)
+	log_admin(text)
+
+/proc/trigger_centcom_recall()
+	if(SSshuttle.emergency.mode != SHUTTLE_CALL)
+		return
+
+	var/time = rand(600, 1200)
```

Comment:
Also while you're at it, I'm sure @nfreader would like to have this stat tracked =^)